README.md

```diff
@@ -12,52 +12,107 @@
+Every pipeline must define exactly 1 Importer, exactly 1 Exporter, and can optionally define a series of 0 or more Processors. A full list of available plugins with `conduit list` and the [plugin documentation page](TODO: plugin docs).
 
-# Contributing
+Here is an example that configures two plugins:
+```yaml
+importer:
+    name: algod
+    config:
+        mode: "follower"
+        netaddr: "http://your-follower-node:1234"
+        token: "your API token"
 
-Contributions are welcome! Please refer to our [CONTRIBUTING](https://github.com/algorand/go-algorand/blob/master/CONTRIBUTING.md) document for general contribution guidelines, and individual plugin documentation for contributing to new and existing Conduit plugins.
+# no processors defined for this configuration
+processors:
+
+exporter:
+    name: "file_writer"
+    config:
+        # the default config writes block data to the data directory.
+```
+
+The `conduit init` command can also be used to select which plugins to include in the template. For example the following command creates a configuration template to populate an Indexer database.
+```sh
+docker run algorand/conduit init --importer algod --processors filter_processor --exporter postgresql > conduit.yml
+```
+
+The config file must be edited before running Conduit.
 
-# Common Setups
+### Run Conduit
 
-The most common usage of Conduit is to get validated blocks from a local `algod` Algorand node, and adding them to a database (such as [PostgreSQL](https://www.postgresql.org/)).
-Users can separately (outside of Conduit) serve that data via an API to make available a variety of prepared queries--this is what the Algorand Indexer does.
+Once configured, start Conduit with your data directory:
+```sh
+./conduit -d data
+```
 
-Conduit works by fetching blocks one at a time via the configured Importer, sending the block data through the configured Processors, and terminating block handling via an Exporter (traditionally a database).
-For a step-by-step walkthrough of a basic Conduit setup, see [Writing Blocks To Files](./docs/tutorials/WritingBlocksToFile.md).
+# Third Party Plugins
+
+<!-- TODO: "Third Party Plugins" or "External Plugins"? -->
+Conduit supports Third Party Plugins, but not in the way you may be used to in other pluggable systems. In order to limit adding dependencies, third party plugins are enabled with a custom build that imports exactly the plugins you would like to deploy.
+
+Over time this process can be automated, but for now it is manual and requires a go development environment and a little bit of code.
+
+For a list of available plugins and instructions on how to use them, see the [Third Party Plugins](./docs/Third_Party_Plugins.md) page.
+
+# Develoment
+
+<!-- TODO: take the first section from docs/Development.md and put it here, similar to what was done with the above section -->
```

Comment:
^^^ TODO for this PR ??? ^^^

Reply:
Yes this is still a draft, there's quite a bit more work to do